detection-rules/link_multifactor_authentication_QR_code_phishing.yml

```diff
@@ -0,0 +1,62 @@
+name: "Link: Multi-Factor Authentication phishing via QR code"
+description: |
+  This rule detects the verbiage related to Multi-factor authentication in the subject, body or attachments. The rule also checks for the presence of a QR code with a URL
+  that contains the recipients email address, from an unsolicited sender.
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound 
+  
+  // body html, plain.raw subject or any attachments contain QR|2fa|MFA|Multi factor authentiation
+  
+  and 1 of (
+      (
+          regex.icontains(subject.subject, '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)')
+      ),
+      (
+          regex.icontains(body.html.display_text, '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)')
+      ),
+      (
+          regex.icontains(body.plain.raw, '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)')
+      ),
+      (
+          any(attachments,
+              .file_type in~ ('bmp', 'png', 'jpg', 'jpeg', 'gif')
+              and any(file.explode(.), 
+                  any(.scan.strings.strings, regex.icontains(.,
+                    '(\b2fa\b|\bQ.?R\.?\s?\b|MFA|Muti[ -]?Factor Auth(entication)?)'))
+              )
+          )
+      )
+  )
+  
+  // QR code is present in attachment & recipient email address is present in the URL, a common tactic used in credential phishing attacks
+  and any(attachments,
+      .file_type in~ ('bmp', 'png', 'jpg', 'jpeg', 'gif')
+          and any(file.explode(.), .scan.qr.data is not null
+          and any(recipients.to, strings.icontains(..scan.qr.data, .email.email))
```

Comment:
indentation is off and I think it's messing up stuff below

Reply:
I've indented and unintended it, it doesn't seem to make any difference in functionality. I'll update so the ands are inline.